desktop/cmp/form/RadioInput.js

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Note: as it stands if you tab into the radio group you can select a button using the arrow keys, but the enter key will not select the highlighted options. Should we change that?